src/common/latest/analyze-task.ts

```diff
@@ -12,9 +11,11 @@ import { EndpointType } from "./sonarqube/Endpoint";
 import Scanner, { ScannerMode } from "./sonarqube/Scanner";
 
 export const analyzeTask: TaskJob = async (endpointType: EndpointType) => {
-  const rootPath = path.join(__dirname, "..");
+  const rootPath = __dirname;
   const jdkVersionSource = tl.getInput("jdkversion", true) as JdkVersionSource;
 
+  tl.warning("THIS SHOULD BE SHOWN");
```

Comment:
purpose?

Reply:
no purpose, it should be dropped, thanks for spotting